crates/tracing/src/lib.rs

```diff
@@ -19,8 +19,8 @@
 //!  fn main() -> eyre::Result<()> {
 //!      let tracer = RethTracer::new().with_stdout(LayerInfo::new(
 //!          LogFormat::Json,
+//!          LevelFilter::INFO.to_string(),
```

Comment:
agreed that info should be default

Reply:
or hang on, what does it mean that info and debug swapped places as `filters` and `directive`?